client/blocks/plugins-update-manager/schedule-form.tsx

```diff
@@ -36,8 +36,8 @@ interface Props {
 }
 export const ScheduleForm = ( props: Props ) => {
 	const { siteSlug } = props;
-	const { data: dataPlugins } = useSitePluginsQuery( siteSlug );
-	const { plugins = [] } = dataPlugins ?? {};
+	const { data } = useCorePluginsQuery( siteSlug, true );
+	const plugins = useMemo( () => data ?? [], [ data ] );
```

Comment:
I don't know if this line makes sense anymore since we move logic of filtering directly into hook.

What do you think?

Reply:
Done!